packages/terra-form-select/src/single/Frame.jsx

```diff
@@ -195,38 +195,11 @@ class Frame extends React.Component {
   /**
    * Opens the dropdown.
    */
-  openDropdown(event) {
+  openDropdown() {
     if (this.state.isOpen || this.props.disabled) {
       return;
     }
 
-    /**
-     * Avoids focusing the input if the toggle button is used to open the select menu.
-     * This is to avoid an issue with VoiceOver on iOS where shifting to toggle button while the
-     * input is focused / onScreen keyboard is open unexpected focus shift when the onScreen
-     * keyboard is closed
-     */
-    if (event && event.target
-      && (event.target.hasAttribute('data-terra-form-select-toggle-button')
```

Comment:
Removing this condition check since data-terra-form-select-toggle-button and data-terra-form-select-toggle-button-icon are not included in single-select.